docs(OFF-872): add doc to use a specific plan presentation (#60)
PrestaShopCorp/docs.cloud.prestashop.com: docs/src/5-prestashop-billing/README.md

  window.psBilling.initializeInvoiceList(window.psBillingContext.context, '#ps-billing-invoice');
  ```

## (Optional) Handle plan selection by yourself

> The billing funnel can handle for you the plan selection, but sometimes you should prefer to present your plan by yourself to be more precise about your offer.

Reviewer: ```suggestion
The billing funnel can handle for you the plan selection, but sometimes you may prefer to present your plan by yourself to be more precise about your offer.
```
should dans ce contexte c'est une recommendation très appuyée, mais pas une obligation, or ici on veut dire au partner que s'il le souhaite, il a cette possibilité.

> In such case you should provide to the billing context the selected plan, and if applicable the quantity associated with.

Reviewer: ```suggestion
In such a case you should provide to the billing context the selected plan, and if applicable the quantity associated with.
```

### Add plan selection

You should add your plan presentation in the configuration page for your module in the back office (located by default at `views/templates/admin/configure.tpl`).

> This is a simple working example, if you need to implement it properly you will have more code to produce.

Reviewer: thought: un peu étrange cette phrase, elle donne l'impression que cette implémentation ne sera pas suffisante dans un cas réel. On pourrait peut-être dire

```suggestion
This is a simple working example that is purposefully basic, you can make the code as complex as you need it to be to cover your needs as long as you follow the steps in this guide.
```

Author: Ouai, en gros ce que je voulais dire c'est que ça ça fonctionne mais en terme d'expérience pour l'utilisateur c'est pourri, mais j'ai pas trop envie d'aller trop loin dans le truc tant qu'on n'a pas quelque chose de plus complet, ie lib de composants des plans et routes qui passent par l'API Gateway

Author: Mais du coup ta proposition me semble bien

    </div>
    <div style="border: 1px solid;margin-left:1rem;padding: 2rem;text-align:center;">
      <h3 style="font-weight: bold;margin-bottom: 1rem;">rbm advanced sandbox</h3>
      <p style="margin-bottom: 1rem;">10€/month</p>
      <button onclick="openCheckout('rbm-advanced')" style="background: black;color: white; padding: 0.5rem; font-weight: bold;">Select this offer</button>
    </div>
    <div style="border: 1px solid;margin-left:1rem;padding: 2rem;text-align:center;">
      <h3 style="font-weight: bold;margin-bottom: 1rem;">rbm ultimate sandbox</h3>
      <p style="margin-bottom: 1rem;">30€/month</p>
      <button onclick="openCheckout('rbm-ultimate')" style="background: black;color: white; padding: 0.5rem; font-weight: bold;">Select this offer</button>
    </div>
  </div>
</section>

<div id="ps-billing-wrapper" style="display:none">
  <div id="ps-billing"></div>
</div>
<div id="ps-modal"></div>

<script src="{$urlAccountsCdn|escape:'htmlall':'UTF-8'}" rel=preload></script>
<script src="{$urlBilling|escape:'htmlall':'UTF-8'}" rel=preload></script>

{literal}

> <script>

Reviewer: thought: Plutôt que de faire un seul bloc avec des explications en commentaires, je "casserais" l'exemple en plusieurs blocs, en expliquant à chaque fois les détails de telles ou telle fonctionnalité qui sont ajoutés dans cet exemple par rapport à l'implémentation basique, un peu sur ce modèle: https://developer.spotify.com/documentation/web-api/tutorials/code-pkce-flow

Author: voilà  j'ai fait une MAJ dis moi ce que tu penses de ça

Reviewer: Yep c'est nickel, le seul truc qui manquerait, c'est un encart avec le fichier où on se trouve, mais Vuepress ne le permet pas nativement je crois

          if(data.subscription) {
            hasSubscription = true;
          }
          break;
        case window.psBilling.EVENT_HOOK_TYPE.SUBSCRIPTION_UPDATED:
          hasSubscription = true;
          break;
      }
      updateBillingCustomerDisplay();
    }

    // Display plan selection or ps billing. In a real case you should do something more complexe to allow your user
    // to change his plan which is not possible in this example.
    function updateBillingCustomerDisplay() {
      if(hasSubscription) {
        document.getElementById('billing-plan-selection').style.display = 'none';
        document.getElementById('ps-billing-wrapper').style.display = 'block';
      } else {
        document.getElementById('billing-plan-selection').style.display = 'block';
        document.getElementById('ps-billing-wrapper').style.display = 'none';
      }
    };

    // Open the checkout full screen modal
    function openCheckout(pricingId) {
      const offerSelection = {offerSelection: {offerPricingId: pricingId }};
      onOpenModal(window.psBilling.MODAL_TYPE.SUBSCRIPTION_FUNNEL, offerSelection);
    };
</script>
{/literal}
```

> ### Retrieve the pricing and pricing id (not available yet)

Reviewer: nitpick: je mettrais les sections en TODO également en commentaire, mieux vaut ne pas afficher quelque chose qui n'est pas complet dans une doc.
